<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
  <head>
    <title>Epigenomics of Common Diseases</title>
    <meta charset="utf-8" />
    <link href="libs/remark-css/default.css" rel="stylesheet" />
    <link href="libs/remark-css/fc.css" rel="stylesheet" />
    <link href="libs/remark-css/fc-fonts.css" rel="stylesheet" />
    <link href="libs/remark-css/hygge.css" rel="stylesheet" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# Epigenomics of Common Diseases
## Wellcome Genome Campus - Hinxton, Cambridge
### 6 - 8 November, 2019<br /> <br/>Jonelle D. Villar, PhD student<br /> Epigenetics and Multiomics Group

---




class: middle, bottom 

background-image: url(img/TandL.jpg), url(img/NORMENT.png), url(img/UiBe.png)
background-position: 50%50%, 95%05%, 59%05%
background-size: 910px, 200px, 275px

#What impressed me most about the trip to Hinxton?

---
class: middle, left

#Great talks about epigenetics&lt;br /&gt; &lt;br /&gt;

.Large[
-  Confounding factors, e.g. circadian rhythms and BMI&lt;br /&gt; &lt;br /&gt;
-  rDNA is sensitive to environmental challenges&lt;br /&gt; &lt;br /&gt;
-  Dysregulated dopamine synthesis in psychosis 
]


---
background-image: url(img/oscC.jpg), url(img/circadian.png)
background-position: 90% 60%, 20% 25%
background-size:500px, 550px

class: Center, top

#Akil Nair, University of Toronto&lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt;


.large[
-  Osc-modC reflects DNAm and demethylation&lt;br /&gt; &lt;br /&gt;
-  Sampling time may be a confounding variable&lt;br /&gt; &lt;br /&gt;
-  Missing link to unexplained epigenomic variation?]

---
background-image:  url(img/pace.jpg), url(img/BMI.png)
background-position: 90% 65%, 30% 35%
background-size: 300px, 600px

class: Center, top

#Stephanie London, NiH, PACE&lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt;

.large[
-  Challenges of interpreting EWAS findings&lt;br /&gt; &lt;br /&gt;
-  Challenges with correcting for smoking and BMI
   ]


---
background-image: url(img/EWAS.png)
background-position: 50% 40%
background-size:600px

class: Center, top

#Catalog search of BMI including *SREBF1*&lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt;
#shows studies of BMI and DNAm as both exposure &amp; outcome.

.large[http://www.ewascatalog.org
]
 
---
class: Center, top

background-image: url(img/variants.jpg), url(img/Rakyan.png) 
background-position: 70% 55%, 25% 25%
background-size:400px, 550px


#Vardhman Rakyan - Queen Mary, University of London&lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt;


.large[
-  rDNA is sensitive to environmental challenges&lt;br /&gt; &lt;br /&gt;
-  Adenine "attracts" DNAm at promoter sites, while cytosine does not&lt;br /&gt; &lt;br /&gt;
-  Linear relationship: reduced weight and CpG-133-A methylation]

---
background-image: url(img/epi_inheritance.png)
background-position: 35% 35%
background-size:700px


class:Center, top  

#Protein restricted (PR) diet&lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt;



.large[
-  Animales with higher % A, silence more A-variants through DNAm at CpG-133&lt;br /&gt; &lt;br /&gt;
-  DNAm pattern maintained into adulthood&lt;br /&gt; &lt;br /&gt;
-  rDNA can display changes to DNA, masquerading as "epigenetic" effects
]


---
background-image: url(img/IGF2.png)
background-position: 35% 30%
background-size:630px

class: Center, top

#Viviane Labrie, Van Andel Institute, USA&lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt;

.large[
-  Hypomethylation of *IGF2*-enhancer ⟹ psychosis-risk&lt;br /&gt; &lt;br /&gt;
-  TH ⟹ rate-limiting enzyme responsible for dopmine synthesis&lt;br /&gt; &lt;br /&gt;
-  Hypomethylation *IGF2* ⟹  risk of SZ, low brain weight (Pidsley *et al* , 2010)  ]



---
background-image: url(img/martens.png), url(img/group2019.jpeg), url(img/NORMENT.png), url(img/UiBe.png)  
background-position: 89%89%, 50%50%, 95%07%, 59%07% 
background-size: 125px, 750px, 200px, 275px

class: Center, top

# **Acknowledgements**&lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&lt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt; &lt;br /&gt;With special thanks to Stéphanie for driving!

---
class: middle, center 
background-image: url(img/Happy2020.jpg)
background-position: cover
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
